client/src/engine/jobs.ts

```diff
@@ -47,8 +47,6 @@ export enum Request {
   lspWorkspaceSymbols = 'lsp/workspaceSymbols',
   lspShowAst = 'lsp/showAst',
 
-  internalRestart = 'ext/restart', // Internal Extension Request
-  internalDownloadLatest = 'ext/downloadLatest', // Internal Extension Request
```

Comment:
We can lose this and still support downloading the latest version, right? You just have to restart VSCode, right?

Reply:
which do you mean? Download compiler or restart?

Reply:
Both, actually

Reply:
Do we need to remove those user facing commands or do they still work?

Reply:
They still work

Reply:
They just don't use the internal commands? OK